projects/angular/src/progress/progress-bars/progress-bar.ts

```diff
@@ -83,6 +95,17 @@ export class ClrProgressBar {
     return this._loop;
   }
 
+  /** @deprecated since 2.0, remove in 4.0 */
```

Comment:
@valentin-mladenov, I think was added by mistake. This does not make sense for a new thing added in v16. Could you open a new PR to remove this line?

Reply:
I expect we need to remove it for `v16` and `next`, rigth?

Reply:
main -> PR #1033

Reply:
@kevinbuhmann We have the design spec, but is not really accessibility oriented. `clrSuccess`, `clrWarning` and `clrDanger` are not really a statuses, but are just colors.
It's marked as depricated as others because of missing accessibility.
Renaming to red, green and yellow is more suitable, in my opinion. I'll raise an accessibility thread about it.

Reply:
Okay.